scripts/windows/installs/vs2013.ps1

```diff
@@ -7,8 +22,11 @@ if ($VerifyHash -ne $FileHash) {
  exit -1
 }
 
+LogWrite "Extracting visual studio"
 cmd /c '7z x "C:\Windows\Temp\vs2013express.iso" -oC:\Windows\Temp\VS2013'
 
+LogWrite "Installing visual studio"
 cmd /c "C:\Windows\Temp\VS2013\wdexpress_full.exe /Passive /NoRestart /Log VisualStudioExpress2013Windows.log"
 
-rm -rf C:\Windows\Temp\VS2013
```

Comment:
This failed for me previously, I assume `rm` isn't available in the context of powershell - but it is available in `cmd`

Reply:
Interesting looking it up `rm` in powershell is commonly an alias for `Remove-Item` and does have different syntax that unix `rm`. I like the move to a more explicit call to `Remove-Item`.